pkg/app/mssql.go

```diff
@@ -0,0 +1,251 @@
+		"\"SELECT name FROM sys.databases WHERE name NOT IN ('master','model','msdb','tempdb')\" -b -s \",\" -h -1"
+
+	loginMssql := []string{"sh", "-c", count}
+	_, stderr, err := m.execCommand(ctx, loginMssql)
+	if err != nil {
+		return errors.Wrapf(err, "Error while Pinging the database: %s", stderr)
+	}
+	log.Print("Ping to the application was success.", field.M{"app": m.name})
+	return err
+}
+
+func (m *MssqlDB) Insert(ctx context.Context) error {
+	log.Print("Adding entry to database", field.M{"app": m.name})
+	insert := "/opt/mssql-tools/bin/sqlcmd -S localhost -U " + m.dbUserName + " -P \"" + m.dbPass + "\" -Q " +
+		"\"USE test; INSERT INTO Inventory VALUES (1, 'banana', 150)\""
+
+	insertQuery := []string{"sh", "-c", insert}
+	_, stderr, err := m.execCommand(ctx, insertQuery)
+	if err != nil {
+		return errors.Wrapf(err, "Error while inserting data into table: %s", stderr)
+	}
+	return err
+}
+
+func (m *MssqlDB) Count(ctx context.Context) (int, error) {
+	log.Print("Counting entries from database", field.M{"app": m.name})
+	insert := "/opt/mssql-tools/bin/sqlcmd -S localhost -U " + m.dbUserName + " -P \"" + m.dbPass + "\" -Q " +
+		"\"SET NOCOUNT ON; USE test; SELECT COUNT(*) FROM Inventory\" -h -1"
+
+	insertQuery := []string{"sh", "-c", insert}
+	stdout, stderr, err := m.execCommand(ctx, insertQuery)
+	if err != nil {
+		return 0, errors.Wrapf(err, "Error while inserting data into table: %s", stderr)
+	}
+	rowsReturned, err := strconv.Atoi(strings.TrimSpace(strings.Split(stdout, "\n")[1]))
+	if err != nil {
+		return 0, errors.Wrapf(err, "Error while converting data: %s", stderr)
+	}
+	return rowsReturned, nil
+}
+
+func (m *MssqlDB) Reset(ctx context.Context) error {
+	log.Print("Reseting database", field.M{"app": m.name})
+	delete := "/opt/mssql-tools/bin/sqlcmd -S localhost -U " + m.dbUserName + " -P \"" + m.dbPass + "\" -Q " +
+		"\"DROP DATABASE test\""
+	deleteQuery := []string{"sh", "-c", delete}
+	_, stderr, err := m.execCommand(ctx, deleteQuery)
+	if err != nil {
+		return errors.Wrapf(err, "Error while inserting data into table: %s", stderr)
+	}
+	return err
+}
+
+func (m *MssqlDB) Initialize(ctx context.Context) error {
+	log.Print("Initializing database", field.M{"app": m.name})
+	createDB := "/opt/mssql-tools/bin/sqlcmd -S localhost -U " + m.dbUserName + " -P \"" + m.dbPass + "\" -Q " +
+		"\"CREATE DATABASE test\""
+
+	createTable := "/opt/mssql-tools/bin/sqlcmd -S localhost -U " + m.dbUserName + " -P \"" + m.dbPass + "\" -Q " +
+		"\"USE test; CREATE TABLE Inventory (id INT, name NVARCHAR(50), quantity INT)\""
+
+	execQuery := []string{"sh", "-c", createDB}
+	_, stderr, err := m.execCommand(ctx, execQuery)
+	if err != nil {
+		return errors.Wrapf(err, "Error while creating the database: %s", stderr)
+	}
+
+	execQuery = []string{"sh", "-c", createTable}
+	_, stderr, err = m.execCommand(ctx, execQuery)
+	if err != nil {
+		return errors.Wrapf(err, "Error while creating table: %s", stderr)
+	}
+	return err
```

Comment:
```suggestion
	return nil
```
[nit] we are already checking error above.